workflow-templates/publish-go-tester-task.yml

```
@@ -39,10 +39,12 @@ jobs:
         id: determination
         run: |
           RELEASE_BRANCH_REGEX="refs/heads/[0-9]+.[0-9]+.x"
+          TAG_REGEX="refs/tags/.*"
           # The `create` event trigger doesn't support `branches` filters, so it's necessary to use Bash instead.
           if [[
             "${{ github.event_name }}" != "create" ||
-            "${{ github.ref }}" =~ $RELEASE_BRANCH_REGEX
+            "${{ github.ref }}" =~ $RELEASE_BRANCH_REGEX && \
+            ! "${{ github.ref }}" =~ $TAG_REGEX \
```

```suggestion
            "${{ github.ref }}" =~ $RELEASE_BRANCH_REGEX &&
            ! "${{ github.ref }}" =~ $TAG_REGEX
```

The newline escaping in these unterminated commands is superfluous. It is present in the workflow in the `arduino/arduino-cli` repo because that workflow never received a sync from this upstream change: https://github.com/arduino/tooling-project-assets/commit/8900f88ad2a5d6e49892857ce9b17ee62200c8b2